subsys/settings/src/settings.c

```diff
@@ -234,39 +242,74 @@ int settings_commit(void)
 	return settings_commit_subtree(NULL);
 }
 
+static bool skip_commit(int ch_cprio, int cprio, int *nprio)
+{
+	if (ch_cprio == cprio) {
+		return false;
+	}
+
+	if (ch_cprio > cprio) {
+		*nprio = (*nprio == cprio) ? ch_cprio : MIN(*nprio, ch_cprio);
+	}
```

Comment:
This is very hard to read - Especially with the short names. 

`cprio` => current prio?
`nprio` => next prio?
`ch_cprio` => `ch` (which is what?) current prio? 

x

Reply:
Ok, the last one is current handler commit priority. I will extend the names.

Reply:
changes applied